backend/src/bin/jobs/downgradeExpiredTrialPlans.ts

```diff
@@ -15,6 +16,7 @@ const job: CrowdJob = {
     console.log('in Downgrade Expired Trial Plans')
```

Comment:
Sorry just noticed - can this be logged with our new logger?

Reply:
done :+1: